checkout_dependencies.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+TFM_DEFAULT_BRANCH="TF-Mv1.1"
+
+usage() { echo "Usage: $0 [-t 'default'|<tfm_versiom>] [-h]" 1>&2; exit 1; }
+
+while getopts ":t:h:" o; do
+    case "${o}" in
+        t)
+            tfm_branch=${OPTARG}
+            ;;
+        *)
+            usage
+            ;;
+    esac
+done
+
+# TFM_DEFAULT_BRANCH if no argument is given or 'default' is asked to checkout
```

Comment:
I don't think this file is required to run the tests in CI as the cloning of test dependencies is done directly in the scripts